arangod/Cluster/ClusterInfo.h

```diff
@@ -1200,7 +1252,13 @@ class ClusterInfo final {
   static constexpr double checkAnalyzersPreconditionTimeout = 10.0;
 
   mutable std::mutex _failedServersMutex;
-  containers::FlatHashSet<ServerID> _failedServers;
+  template<typename K>
+  using NodeHashSet =
+      containers::NodeHashSet<K, typename containers::NodeHashSet<K>::hasher,
+                              typename containers::NodeHashSet<K>::key_equal,
+                              ResourceUsageAllocator<K>>;
+
+  NodeHashSet<pmr::ServerID> _failedServers;
```

Comment:
Previously `_failedServers` was a FlatHashSet. Now it is a NodeHashSet. Which is fine, but is there a specific reason for this change?

Reply:
It's maybe because increased size of entry, in arangodb we have static asserts to prevent such usage of absl hash tables.
40 bytes now.

I think speed is pretty same in such case according to benchmark, so we probably could increase the limit, or just use node hash set here as Lars wrote

Reply:
Personally me prefer increase limit to 40, 32 was written because string is 32.
But for maps we use bigger pair sizeof, so I think it's ok to just change this to 40

Reply:
Discussed with @MBkkt : can be ignored for now.